src/site/content/en/fast/native-lazy-loading/index.md

```diff
@@ -132,7 +130,7 @@ The best practice of setting dimensions applies to `<img>` tags regardless of wh
 
 Images will still lazy-load if dimensions are not included, but [specifying them decreases the chance of layout shift](https://www.youtube.com/watch?v=4-d_SoCHeWE). If you are unable to include dimensions for your images, lazy-loading them can be a trade-off between saving network resources and potentially being more at risk of layout shift. 
 
-While native lazy-loading in Chromium is implemented in a way such that images are likely to be loaded once they are visible, it should be kept in mind, there is a slightly greater chance of them not being loaded yet. In this case, case missing `width` and `height` attributes on such images increase their impact on Cumulative Layout Shift.
+While native lazy-loading in Chromium is implemented in a way such that images are likely to be loaded once they are visible, it should be kept in mind, there is a slightly greater chance of them not being loaded yet. In this case, missing `width` and `height` attributes on such images increase their impact on Cumulative Layout Shift.
```

Comment:
This isn't related to this PR, but "there is a slightly greater chance of them not being loaded yet" makes it sound like there is a greater chance that they won't be loaded _compared to_ them being loaded. Is that true? I feel like the intention of this sentence is to say that "there is still a small chance that they might not be loaded yet [...]". Thoughts?

Reply:
The wording here could certainly be improved. I actually like your proposed version of this sentence. I'm going to go ahead and integrate this into the sentence as it's worded more clearly. Thanks, Dom!